digits/extensions/view/interface.py

```diff
@@ -49,6 +49,32 @@ def get_header_template(self):
         """
         return None, None
 
+    def get_ng_templates(self):
```

Comment:
Actually, @gheinrich, if I move the app as a parent to the header, I can add a div in the header_html and is closed in the footer_html. That should work and clean things up a bit.